apps/coscrad-frontend/src/components/nav-menu/nav-menu-presenter.tsx

```diff
@@ -37,6 +38,8 @@ export const NavMenuPresenter = ({ navItemInfos }: NavMenuPresenterProps): JSX.E
                 </IconButton>
             </Tooltip>
             <AuthenticationButton />
+            {/* TODO We should present the `CurrentUserInfo` below the menu and login icons  */}
```

Comment:
The  `CurrentUserInfo` should be inline on the left-hand side of the menu and login buttons

Reply:
Do you have a recommendation for how to do this? I'm sure it's obvious, but I'm a back-end guy.

Reply:
You did this for me.